### PR TITLE
Force update

### PR DESCRIPTION
**Pull request by coot**
[forceUpdate](https://facebook.github.io/react/docs/react-component.html#forceupdate). This PR includes two versions: with and without the callback argument:
```
forceUpdate :: forall eff props state. ReactThis props state -> Eff eff Unit

forceUpdateCb :: forall eff props state. ReactThis props state -> Eff eff Unit -> Eff eff Unit
```

### CODE REVIEW

File: src/React.purs [hunk 1]
```diff
@@ -335,6 +340,25 @@ createClassStateless' k =
   createClassStateless \props ->
     k props (childrenToArray (unsafeCoerce props).children)
 
+foreign import forceUpdateImpl :: forall eff props state.
```
Comment (paf31) [inline src/React.purs]: Can this one can be implemented  in terms of the other using `pure unit`?

File: src/React.js [hunk 2]
```diff
@@ -122,6 +122,20 @@ function createClass(spec) {
 }
 exports.createClass = createClass;
 
+function forceUpdateImpl(this_) {
```
Comment (paf31) [inline src/React.js]: This is no longer used, right?

Comment (coot) [inline src/React.js]: Indeed.

Comment (coot) [inline src/React.js]: I just forgot to push all changes.

**Comment by ethul** [comment]
Looks good to me. @paf31 Good for me to do the merge?

**Comment by paf31** [comment]
👍 If you're happy, go ahead please. Sorry I haven't had time to review PRs as much lately.

**Comment by ethul** [comment]
No problem. Thanks!

**Pull request merged**